File: 8_Kyu_Kata/counting_sheep_dot_dot_dot/Python/counting_sheep_dot_dot_dot.py
# 8 kyu - Counting sheep...  [ ID: 54edbc7200b811e956000556  (counting-sheep-dot-dot-dot) ]
# URL: https://www.codewars.com/kata/54edbc7200b811e956000556
# Category: NONE  |  Tags: ARRAYS | FUNDAMENTALS
# ******************************************************************************
def count_sheeps(sheep):
    return sum(present is True for present in sheep)

File: 8-kyu-kata/counting-sheep-dot-dot-dot/javascript/countingSheepDotDotDot.js
// 8 kyu - Counting sheep...  [ ID: 54edbc7200b811e956000556  (counting-sheep-dot-dot-dot) ]
// URL: https://www.codewars.com/kata/54edbc7200b811e956000556
// Category: REFERENCE  |  Tags: ARRAYS | FUNDAMENTALS
// *****************************************************************************
function countSheeps(arrayOfSheep) {
    return arrayOfSheep.reduce((a,b) => b ? a + 1 : a, 0)
  }
